privacyidea/api/before_after.py

```diff
@@ -469,6 +469,11 @@ def after_request(response):
     This function is called after a request
     :return: The response
     """
+    # close any open ldap connection
+    store = get_request_local_store()
+    resolvers = store.get("resolver_objects", {})
+    for resolver in resolvers.values():
+        resolver.unbind()
```

Comment:
In `lifecycle.py` we implemented finalizers for this task:
https://github.com/privacyidea/privacyidea/blob/8c6813775f9b3c4f140c55892f61ffd920f00257/privacyidea/lib/lifecycle.py#L25
They are used in the `SQLIdResolver.py` to close the db session:
https://github.com/privacyidea/privacyidea/blob/db3f1ace8ce0a59918e3440719556bfbaae857f0/privacyidea/lib/resolvers/SQLIdResolver.py#L497
The finalizers are called when the request is torn down:
https://github.com/privacyidea/privacyidea/blob/c2158e37e5fa8397c02a7aae1d14527951d0817f/privacyidea/api/before_after.py#L104
We could set the finalizer to `self.connection.unbind`.